src/main/resources/docker.gradle

```diff
@@ -1,15 +1,28 @@
 apply plugin: 'com.google.cloud.tools.jib'
 
 
+ext.projectName = project.name.toLowerCase()
```

Comment:
we don't need this as a global variable. Maybe it should be inside the function below, or you can even quote it inside the string:
```
"${registry}${project.name.toLowerCase()}:${version}"
```

Reply:
done